lib/msal-common/src/authority/Authority.ts

```diff
@@ -438,33 +443,46 @@ export class Authority {
      * @param newMetadata
      */
     private async updateCloudDiscoveryMetadata(metadataEntity: AuthorityMetadataEntity): Promise<AuthorityMetadataSource> {
+        this.logger.verbose("Attempting to get cloud discovery metadata in the config");
```

Comment:
Can we log the user's authority somewhere here?

Reply:
I don't understand - what specifically do you want to log?